#######################################################################################################################
################################################### CAMERA PARAMETERS #################################################
#######################################################################################################################

desired_camera_name:

  camera_intrinsic:           # PINHOLE CAMERA INTRINSICS
    fx: 700.0
    fy: 700.0
    ccx: 340.0
    ccy: 250.0

  distortion_coefficient:     # RADIAL AND TANGENTIAL DISTORTION COEFFICIENTS
    radial:
      k1: 0.1
      k2: 0.3
    tangential:
      p1: 0.0
      p2: 0.0

  body_camera_matrix:         # ROTATION MATRIX BETWEEN BODY AND CAMERA FRAME
     rows: 3
     cols: 3
     data: [1, 0, 0, 0, 1, 0, 0, 0, 1]


#######################################################################################################################
#######################################################################################################################
#######################################################################################################################



